<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>affineWindow</class>
 <widget class="QDialog" name="affineWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1050</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>input key</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sources.qrc">
    <normaloff>:/Images/window icon.png</normaloff>:/Images/window icon.png</iconset>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>560</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="sources.qrc">:/Images/Affine Cipher input.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="aNumber">
   <property name="geometry">
    <rect>
     <x>306</x>
     <y>118</y>
     <width>630</width>
     <height>57</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit{
color:rgb(28, 79, 130);
padding-top:10px;
	font: 700 12pt &quot;Segoe UI&quot;;
backgroudn:transparent;border:none;
}</string>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="bNumber">
   <property name="geometry">
    <rect>
     <x>306</x>
     <y>219</y>
     <width>630</width>
     <height>57</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit{
color:rgb(28, 79, 130);
padding-top:10px;
	font: 700 12pt &quot;Segoe UI&quot;;
backgroudn:transparent;border:none;
}</string>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="cNumber">
   <property name="geometry">
    <rect>
     <x>306</x>
     <y>317</y>
     <width>630</width>
     <height>57</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit{
color:rgb(28, 79, 130);
padding-top:10px;
	font: 700 12pt &quot;Segoe UI&quot;;
backgroudn:transparent;border:none;
}</string>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="doneBtn">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>405</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:transparent;border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="sources.qrc"/>
 </resources>
 <connections/>
</ui>
